{
	"document": {
		"aggregate_severity": {
			"namespace": "https://www.suse.com/support/security/rating/",
			"text": "low"
		},
		"category": "csaf_vex",
		"csaf_version": "2.0",
		"distribution": {
			"text": "Copyright 2024 SUSE LLC. All rights reserved.",
			"tlp": {
				"label": "WHITE",
				"url": "https://www.first.org/tlp/"
			}
		},
		"lang": "en",
		"notes": [
			{
				"category": "summary",
				"text": "SUSE CVE-2023-26303",
				"title": "Title"
			},
			{
				"category": "description",
				"text": "Denial of service could be caused to markdown-it-py, before v2.2.0, if an attacker was allowed to force null assertions with specially crafted input.",
				"title": "Description of the CVE"
			},
			{
				"category": "legal_disclaimer",
				"text": "CSAF 2.0 data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
				"title": "Terms of use"
			}
		],
		"publisher": {
			"category": "vendor",
			"contact_details": "https://www.suse.com/support/security/contact/",
			"name": "SUSE Product Security Team",
			"namespace": "https://www.suse.com/"
		},
		"references": [
			{
				"category": "external",
				"summary": "CVE-2023-26303",
				"url": "https://www.suse.com/security/cve/CVE-2023-26303"
			},
			{
				"category": "external",
				"summary": "SUSE Security Ratings",
				"url": "https://www.suse.com/support/security/rating/"
			},
			{
				"category": "external",
				"summary": "SUSE Bug 1208974 for CVE-2023-26303",
				"url": "https://bugzilla.suse.com/1208974"
			}
		],
		"title": "SUSE CVE CVE-2023-26303",
		"tracking": {
			"current_release_date": "2023-03-10T04:02:27Z",
			"generator": {
				"date": "2023-03-07T03:13:16Z",
				"engine": {
					"name": "cve-database.git:bin/generate-csaf-vex.pl",
					"version": "1"
				}
			},
			"id": "CVE-2023-26303",
			"initial_release_date": "2023-03-07T03:13:16Z",
			"revision_history": [
				{
					"date": "2023-03-07T03:13:16Z",
					"number": "2",
					"summary": "Current version"
				},
				{
					"date": "2023-03-10T04:02:27Z",
					"number": "3",
					"summary": "Current version"
				}
			],
			"status": "interim",
			"version": "3"
		}
	},
	"product_tree": {
		"branches": [
			{
				"branches": [
					{
						"branches": [
							{
								"category": "product_name",
								"name": "openSUSE Tumbleweed",
								"product": {
									"name": "openSUSE Tumbleweed",
									"product_id": "openSUSE Tumbleweed",
									"product_identification_helper": {
										"cpe": "cpe:/o:opensuse:tumbleweed"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python310-markdown-it-py-2.2.0-1.1",
								"product": {
									"name": "python310-markdown-it-py-2.2.0-1.1",
									"product_id": "python310-markdown-it-py-2.2.0-1.1"
								}
							},
							{
								"category": "product_version",
								"name": "python311-markdown-it-py-2.2.0-1.1",
								"product": {
									"name": "python311-markdown-it-py-2.2.0-1.1",
									"product_id": "python311-markdown-it-py-2.2.0-1.1"
								}
							},
							{
								"category": "product_version",
								"name": "python38-markdown-it-py-2.2.0-1.1",
								"product": {
									"name": "python38-markdown-it-py-2.2.0-1.1",
									"product_id": "python38-markdown-it-py-2.2.0-1.1"
								}
							},
							{
								"category": "product_version",
								"name": "python39-markdown-it-py-2.2.0-1.1",
								"product": {
									"name": "python39-markdown-it-py-2.2.0-1.1",
									"product_id": "python39-markdown-it-py-2.2.0-1.1"
								}
							}
						],
						"category": "product_family",
						"name": "SUSE Linux Enterprise"
					}
				],
				"category": "vendor",
				"name": "SUSE"
			}
		],
		"relationships": [
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python310-markdown-it-py-2.2.0-1.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python310-markdown-it-py-2.2.0-1.1"
				},
				"product_reference": "python310-markdown-it-py-2.2.0-1.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python311-markdown-it-py-2.2.0-1.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python311-markdown-it-py-2.2.0-1.1"
				},
				"product_reference": "python311-markdown-it-py-2.2.0-1.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python38-markdown-it-py-2.2.0-1.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python38-markdown-it-py-2.2.0-1.1"
				},
				"product_reference": "python38-markdown-it-py-2.2.0-1.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python39-markdown-it-py-2.2.0-1.1 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python39-markdown-it-py-2.2.0-1.1"
				},
				"product_reference": "python39-markdown-it-py-2.2.0-1.1",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			}
		]
	},
	"vulnerabilities": [
		{
			"cve": "CVE-2023-26303",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2023-26303"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "Denial of service could be caused to markdown-it-py, before v2.2.0, if an attacker was allowed to force null assertions with specially crafted input.",
					"title": "CVE description"
				}
			],
			"product_status": {},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2023-26303",
					"url": "https://www.suse.com/security/cve/CVE-2023-26303"
				},
				{
					"category": "external",
					"summary": "SUSE Security Ratings",
					"url": "https://www.suse.com/support/security/rating/"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1208974 for CVE-2023-26303",
					"url": "https://bugzilla.suse.com/1208974"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:python310-markdown-it-py-2.2.0-1.1",
						"openSUSE Tumbleweed:python311-markdown-it-py-2.2.0-1.1",
						"openSUSE Tumbleweed:python38-markdown-it-py-2.2.0-1.1",
						"openSUSE Tumbleweed:python39-markdown-it-py-2.2.0-1.1"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 3.3,
						"baseSeverity": "LOW",
						"vectorString": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:L",
						"version": "3.1"
					},
					"products": [
						"openSUSE Tumbleweed:python310-markdown-it-py-2.2.0-1.1",
						"openSUSE Tumbleweed:python311-markdown-it-py-2.2.0-1.1",
						"openSUSE Tumbleweed:python38-markdown-it-py-2.2.0-1.1",
						"openSUSE Tumbleweed:python39-markdown-it-py-2.2.0-1.1"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2023-02-23T02:00:43Z",
					"details": "low"
				}
			],
			"title": "CVE-2023-26303"
		}
	]
}
